packages/cloudflare/src/cli/templates/worker.ts

```diff
@@ -42,7 +42,7 @@ export default {
           .default as typeof import("next/dist/server/next-server").default;
 
         requestHandler = new NextNodeServer({
-          conf: { ...nextConfig, env },
+          conf: nextConfig,
```

Comment:
note: nextconfig env option is for inlining variables during build. that isn't applicable for us here.